packages/gatsby-theme-docs/src/layouts/release-notes-detail.js

```diff
@@ -50,24 +50,13 @@ const LayoutReleaseNotesDetail = (props) => {
                     font-size: ${designSystem.typography.fontSizes.small};
                   `}
                 >
-                  {'Back to all releases'}
+                  {`All Release Notes`}
                 </span>
               </Link>
             </SpacingsInline>
-            <Markdown.H1
-              // Use h1 for the page title but style it as an h3.
-              css={css`
-                font-size: ${designSystem.typography.fontSizes.h3};
-                margin: ${designSystem.dimensions.spacings.big} 0 0;
-                color: unset;
-              `}
-            >
-              {props.pageData.title}
-            </Markdown.H1>
+            <ReleaseNote.H1>{props.pageData.title}</ReleaseNote.H1>
```

Comment:
In this case you should be able to do `<Markdown.H1 as="h3"`

https://emotion.sh/docs/styled#as-prop